Tests/UnitTests/Scripts/tests.ts

```diff
@@ -567,7 +582,7 @@ describe("URL", function () {
     it("should update href after URLSearchParams are changed", function () {
         // Augment URL with pathname and search
         const url = new URL("https://httpbin.org/en-US/docs?foo=1&bar=2");
-        url.searchParams.set("foo", 999);
+        url.searchParams.set("foo", "999");
```

Comment:
Should this be the same as the others where you cast to `any`? This is semantically different by passing in a string.